### PR TITLE
[prometheus-smartctl-exporter]: expose serviceMonitor `interval` and `scrapeTimeout`

### PR DESCRIPTION
**Pull request by yellowhat**
#### What this PR does / why we need it

* expose serviceMonitor `interval` option
* expose serviceMonitor `scrapeTimeout` option

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-smartctl-exporter]`)

### CODE REVIEW

**Review by kfox1111 — APPROVED** [review]
lgtm

**Comment by yellowhat** [comment]
@kfox1111 may I ask for a review?

Thanks

**Review by kfox1111 — APPROVED** [review]
lgtm

**Comment by yellowhat** [comment]
@kfox1111 may I ask for an approval?

Thanks

**Review by kfox1111 — DISMISSED** [review]
lgtm

**Comment by kfox1111** [comment]
Not sure I have enough permissions to do so. I tried.

**Comment by yellowhat** [comment]
It is still "Review required"

**Comment by yellowhat** [comment]
@kfox1111 can you suggest who can I tag to approve it?

Thanks

**Comment by kfox1111** [comment]
I think @zanhsieh approved your last one.

**Review by kfox1111 — DISMISSED** [review]

**Review by zanhsieh — APPROVED** [review]

File: charts/prometheus-smartctl-exporter/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -5,22 +5,22 @@ metadata:
   name: {{ template "prometheus-smartctl-exporter.fullname" . }}
   labels:
     {{- include "prometheus-smartctl-exporter.labels" . | nindent 4 }}
-{{- if ne (len .Values.serviceMonitor.extraLabels) 0 }}
-{{ toYaml .Values.serviceMonitor.extraLabels | indent 4 }}
-{{- end }}
+    {{- if ne (len .Values.serviceMonitor.extraLabels) 0 }}
+    {{ toYaml .Values.serviceMonitor.extraLabels | indent 4 }}
+    {{- end }}
```
Comment (zanhsieh) [inline charts/prometheus-smartctl-exporter/templates/servicemonitor.yaml:10]: Why not go with `{{ with }}` ?
```suggestion
    {{- with .Values.serviceMonitor.extraLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Comment by yellowhat** [comment]
@zanhsieh What about now?

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**